docs/developer/channels.mdx

```diff
@@ -253,6 +271,104 @@ Response:
 }
 ```
 
+## Reorder warehouses within channels
+
+The warehouses assigned to the channel can be sorted. The provided order defines the
+warehouses' order used in `PRIORITIZE_SORTING_ORDER` allocation strategy.
+The `sort_order` in `moves` input represents the new relative position of the item.
+So when 1 is provided the item will be moved one position forward, when -1 - the one
+position backward.
+
+Let's assume that we have a `channel` with three channels in the following order.
+
+```json
+{
+  "data": {
+    "channel": {
+      "id": "Q2hhbm5lbDox",
+      "warehouses": [
+        {
+          "id": "V2FyZWhvdXNlOjU1NTZiOWI0LTc1ZTItNGI3YS1hZWM1LTQxOTY4NDA2OGE4OA==",
+          "slug": "europe"
+        },
+        {
+          "id": "V2FyZWhvdXNlOjQwZWY1MTQwLWQ5OTYtNDVlNy04NzUzLTlkZThkMTdhMjg1Yw==",
+          "slug": "oceania"
+        },
+        {
+          "id": "V2FyZWhvdXNlOjY4M2FkMzZhLTRmNjktNDI2ZS1iYzUyLTMyZGJiZTQ2NjUyZA==",
+          "slug": "americas"
+        }
+      ]
+    }
+  }
+}
+```
+
+To move the `americas` warehouse to the first place and the `europe` warehouse to
+third, we can run the following mutation.
+
+```graphql
+mutation {
+  channelReorderWarehouses(
+    channelId: "Q2hhbm5lbDox"
+    moves: [
+      {
+        id: "V2FyZWhvdXNlOjY4M2FkMzZhLTRmNjktNDI2ZS1iYzUyLTMyZGJiZTQ2NjUyZA=="
```

Comment:
maybe a comment in the payload that this is `america` and the other one is `europe` would make it a bit easier to follow